CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ### 0.9.2 (Next)
 
+* [#169](https://github.com/slack-ruby/slack-ruby-client/pull/169): Updated to latest slack-api-ref; adds [Conversations API](https://api.slack.com/docs/conversations-api) and support for locale; **removes `groups_close`** - [@jmanian](https://github.com/jmanian).
```

Comment:
We usually just call these things as separate line items, no need to bold anything.

So 

```
[#169](https://github.com/slack-ruby/slack-ruby-client/pull/169): Added [Conversations API](https://api.slack.com/docs/conversations-api) - [@jmanian](https://github.com/jmanian).
[#169](https://github.com/slack-ruby/slack-ruby-client/pull/169): Removed `groups_close` - [@jmanian](https://github.com/jmanian).
```